eZ/Bundle/EzPublishCoreBundle/DependencyInjection/Configuration.php

```diff
@@ -124,6 +124,10 @@ function ($v) {
                                         $v['fields_groups']['default'] = 'content';
                                     }
 
+                                    if (!isset($v['options'])) {
+                                        $v['options'] = array();
```

Comment:
Nitpick: I've already seen in this PR short array syntax, so for consistency: `$v['options'] = [];`